pkg/metrics/features/metrics.go

```diff
@@ -36,6 +36,7 @@ type Metrics struct {
 	ACLBBGPAdvertisementEnabled     metric.Gauge
 	ACLBEgressGatewayEnabled        metric.Gauge
 	ACLBBandwidthManagerEnabled     metric.Gauge
+	ACLBSRv6Enabled                 metric.Gauge
```

Comment:
This feature is hidden from users today because we only have the datapath. The agent flag for instance is hidden and we only use it for tests. So, we probably don't want to add it to metrics.